app/src/main/res/values-da/strings.xml

```diff
@@ -636,11 +628,24 @@
     <!-- This string is displayed as the title of the Bookmarks List, which contains the user's bookmarked websites. -->
     <string name="bookmarks_title">Bogmærker</string>
 
+    <!-- This string is displayed as the title of the Bookmarks List when the bookmarks list is empty. -->
+    <string name="bookmarks_empty">Din liste med bogmærker er tom</string>
+
+    <!-- This string is displayed under the bookmarks title in the bookmarks list. -->
+    <string name="bookmarks_description">Klik på @-ikonet for at tilføje et bogmærke til den side, du er på.</string>
```

Comment:
Is the "-" specific to some languages? I see that most of them don't use it but some do.